### ISSUE TITLE
i386: Parity Flag not modeled for some instructions

### ISSUE THREAD

**Issue by rfalke** [NationalSecurityAgency/ghidra#2223]
**Describe the bug**
Output contains a magic `in_PF` but doesn't define it.

**To Reproduce**
1. decompile https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/stress_int_ia32/ia32_elf_base_inst__1_var__no_complex/subject.exe
2. Output is:
```
undefined8 inst_0_flags_var_0(void)

{
  undefined4 in_EDX;
  int iVar1;
  bool in_PF;
  
  iVar1 = 0;
  if (in_PF) {
    iVar1 = 4;
  }
  return CONCAT44(in_EDX,iVar1 + -4);
}
```

The `adc` instruction sets it in this case. There are more cases for other instructions. Search in https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/stress_int_ia32/ia32_elf_base_inst__1_var__no_complex/by_ghidra.c for "in_PF".

**Expected behavior**
Output should be:
```
int32_t inst_0_flags_var_0(void)

{ 
  return 0;
}
```

**Environment (please complete the following information):**
 - OS: Linux
 - Java Version: openjdk version "11.0.5" 2019-10-15
 - Ghidra Version: 9.1.2_PUBLIC

**Additional context**
- Background: http://kronotai.com/wordpress/339/testing-decompiler-semantics/

**Comment by astrelsky** [comment]
This has been fixed in 9.2.

**Comment by rfalke** [comment]
As far as I see 9.2 isn't released yet?!

**Comment by ryanmkurtz** [comment]
Anything in the master branch will end up in 9.2 when it gets released.  So we usually just call the current master branch 9.2 (technically it's 9.2-DEV).

**Comment by rfalke** [comment]
So far I haven't tried compiling ghidra myself since it is unknown how much effort it required. I will try when I have some time.

Are there any release plans for 9.2?

**Comment by astrelsky** [comment]
> So far I haven't tried compiling ghidra myself since it is unknown how much effort it required. I will try when I have some time.
> 
> Are there any release plans for 9.2?

You can get a pre-built one [here](https://github.com/garyttierney/ghidra-batteries-included/actions/runs/212796178) if you'd like.

**Comment by rfalke** [comment]
This is no longer an issue in ghidra_10.3.2_PUBLIC.
